pkg/aws/helpers.go

```diff
@@ -762,3 +762,12 @@ func FindOperatorRoleBySTSOperator(operatorRoles []*cmv1.OperatorIAMRole, operat
 	}
 	return ""
 }
+
+func GetSTSPolicyDetails(policies map[string]*cmv1.AWSSTSPolicy, key string) string {
```

Comment:
This function preserves the current behavior, as we used a map of `[string]string` for policies.
If a key isn't in the map, the value retrieved from the map is `""`.